app/boards/shields/leeloo/leeloo_common.dtsi

```diff
@@ -50,22 +50,21 @@ RC(3,0) RC(3,1) RC(3,2) RC(3,3) RC(3,4) RC(3,5) RC(4,5) RC(4,6) RC(3,6) RC(3,7)
         label = "LEFT_ENCODER";
         a-gpios = <&pro_micro 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&pro_micro 20 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <2>;
-        status = "disabled";
+        steps = <80>;
     };
 
     right_encoder: right_encoder {
         compatible = "alps,ec11";
         label = "RIGHT_ENCODER";
         a-gpios = <&pro_micro 20 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&pro_micro 21 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <2>;
-        status = "disabled";
+        steps = <80>;
     };
 
     sensors {
         compatible = "zmk,keymap-sensors";
         sensors = <&left_encoder &right_encoder>;
+        triggers-per-rotation = <30>;
```

Comment:
I think to be equivalent to the previous resolution value of `2`, this should be set `40` since `steps` is `80`. In practice it might have the same effect as `30` but I believe `steps` should always be divisible by `triggers-per-rotation`.

(same comment for leeloo_micro)

Reply:
Hi @caksoylar,

I was able to do some more research on the rotary encoders that I'm using. Based on the data sheet, the pulses are 15 and the detents are 30. I've adjusted configuration items in the .dtsi files of Leeloo and Leeloo-Micro and they work well. But, not at a reasonable resolution. For each detent, there is an increase of 8 (i.e., 0, 8, 16, etc). When I change the pulses to 120, it changes the resolution to 2 (i.e., 0, 2, 4, etc). This makes each detent feel more predictable. I've set both .dtsi files to 120/30.

Thank you!

:)

Reply:
If it works for your hardware, good enough for me :)

Reply:
Thank you again for all your work in reviewing and helping me out @caksoylar!